docker/docker-compose-kvalobs.yml

```diff
@@ -26,7 +26,14 @@ services:
       POSTGRES_PASSWORD: postgres
     volumes:
       - kvpostgres:/var/lib/postgresql/data
-
+  
+  kvdataiputd:
+    image: focal-kvdatainputd:latest
+    ports: 
+      - '8090:8090'
+    volumes:
```

Comment:
Maybe not use local paths like this? It makes the file useless to anyone but borgem.